### DISCUSSION TITLE
Welcome to DevTools Discussions!

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#2117] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Aryangp** [comment]
Hi I am a MERN stack developer wanting to do open source contribution and i came across replay.io devtools  and i have a question is there is no issues tab in this repository so how to go around that @jasonLaster
